presto-main/src/main/java/com/facebook/presto/sql/planner/sanity/PlanChecker.java

```diff
@@ -47,7 +47,6 @@ public PlanChecker(FeaturesConfig featuresConfig, boolean forceSingleNode)
                         new NoDuplicatePlanNodeIdsChecker(),
                         new TypeValidator(),
                         new NoSubqueryExpressionLeftChecker(),
-                        new NoIdentifierLeftChecker(),
```

Comment:
Why are we removing a plan sanity checker?

Reply:
woops, will fix it